src/content/en/updates/posts/2016/05/houdini.md

```diff
@@ -0,0 +1,366 @@
+---
+layout: updates/post
+title: "Houdini – Demystifying CSS"
+description: "Houdini is a collection of APIs that expose the CSS engine’s internals to developers"
+published_on: 2016-05-20
+updated_on: 2016-05-20
+authors:
+  - surma
+tags:
+  - houdini
+  - css
+featured_image: /web/updates/images/2016/05/houdini/compworklet_small.png
+
+---
+
+Have you ever thought about the amount of work CSS does? You change a single
+attribute and suddenly your entire website appears in a different layout. It’s
+kind of *magic* in that regard (can you tell where I am going with this?!). So
+far, we – the community of web developers – have only been able to witness and
+observe the magic. What if we want to come up with our own magic tricks and
+perform them? What if we want to *become the magician*? Enter Houdini!
+
+The Houdini task force consists of engineers from Mozilla, Apple, Opera,
+Microsoft, HP, Intel and Google working together to expose certain parts of the
+CSS engine to web developers. The task force is working on a *collection of
+drafts* with the goal to get them accepted by the W3C to become actual web
+standards. They set themselves a few high-level goals, turned them into
+specification drafts which in turn gave birth to a set of supporting,
+lower-level specification drafts. The combination of these drafts is what is
+usually meant when someone is talking about “Houdini”. At the time of writing,
+the [list of drafts][Houdini Drafts] is still incomplete and some of the drafts
+are mere placeholders. That’s how early in development of Houdini we are.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/EUlIxr8mk7s" frameborder="0" allowfullscreen></iframe>
+
+*Disclaimer:* I want to give a quick overview of the Houdini drafts so you have
+*an idea of what kind of problems Houdini tries to tackle. As far as the current
+*state of the specs allow, I’ll try to give code examples, as well. Keeping that
+*in mind, please be aware that all of these specs are *drafts* and very
+*volatile. There’s no guarantee that these code samples are even remotely
+*correct in the future or that any of these drafts become reality at all.
+
+## The specifications
+### Worklets ([spec][Worklets spec])
```

Comment:
The `###` isn't being properly interpreted by the markdown renderer:
![image](https://cloud.githubusercontent.com/assets/1749548/15082762/858818ec-1397-11e6-8615-f434070c538b.png)